wiki/Debugging.md

```diff
@@ -61,7 +60,8 @@
     ```
 
 ### Using FBCon
-FBCon enables kernel output to the display. Use this as a last resort if the above mentioned methods dont work
+
+FBCon enables kernel output to the display. Use this as a last resort if the above mentioned methods don't work.

```

Comment:
Note: if you don't select these options from menu configuration it won't work

I'm not near a pc to tell the menu configuration entries

But if you can change this to what all options in menuconfig I'll be grateful

Reply:
about this one... uhh, i dont have any pc rn and i dont have kernel source cloned on my phone, so cant :sorry:

Reply:
It's alright